tools/pubsys/src/main.rs

```diff
@@ -38,8 +38,7 @@ fn run() -> Result<()> {
     // Parse and store the args passed to the program
     let args = Args::from_args();
 
-    // TerminalMode::Mixed will send errors to stderr and anything less to stdout.
-    TermLogger::init(args.log_level, LogConfig::default(), TerminalMode::Mixed)
```

Comment:
From what I've read, pubsys should always run in an environment where TTY is available, so I'm a bit unsure whether I should update it to be consistent with the rest of simplelog usage. For now I did it anyway but let me know if this file should not be changed.

Reply:
We currently expect pubsys will be run interactively, but I don't think we get much benefit from terminal coloring on this, and I wouldn't want to block interesting noninteractive uses.  So, I think this is probably right.

Reply:
Sweet!